routes/soldiers.js

```diff
@@ -0,0 +1,75 @@
+import events from 'events';
+import { connectToDB, client } from '../connections.js';
```

Comment:
dont need `connectToDB` - this is already in `loaders`

Reply:
fixed